public/app/features/admin/state/actions.ts

```diff
@@ -138,3 +152,19 @@ export function revokeAllSessions(userId: number): ThunkResult<void> {
     dispatch(loadUserSessions(userId));
   };
 }
+
+// UserAdminPage
+
+export function loadUserProfile(userId: number): ThunkResult<void> {
```

Comment:
Thunks can be tested using [thunkTester](https://github.com/grafana/grafana/blob/master/contribute/style-guides/redux.md#thunktester) and you should consider adding tests for these thunks.